.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+group: edge
+sudo: required
+services:
+  - mongodb
+  - docker
+node_js:
+  - "0.10"
+  - "4.4.3"
+before_install: npm install
```

Comment:
Shouldn't be the other way arround? on before_install install dependencies and on install just `npm install`

```
before_install:
  - npm install fh-build -g
  - fh-build template
install: npm install
```